build.gradle

```diff
@@ -59,7 +58,7 @@ allprojects {
     // with Navigation Native forced to be some snapshot version
     // if you need to use snapshots while development, just set `addSnapshotsRepo` to true manually
     def addSnapshotsRepo = project.hasProperty('ALLOW_SNAPSHOT_REPOSITORY') ? project.property('ALLOW_SNAPSHOT_REPOSITORY') : (System.getenv("ALLOW_SNAPSHOT_REPOSITORY")?.toBoolean() ?: false)
-    if (addSnapshotsRepo) {
+    if (true) { // todo revert after bumping mapbox-java to pre release
```

Comment:
This will be reverted and replaced with a `mapbox-java` version bump before merging.